helm/cluster-cloud-director/templates/cloud-provider-cloud-director-helmrelease.yaml

```diff
@@ -24,10 +24,10 @@ spec:
       chart: cloud-provider-cloud-director
       # used by renovate
       # repo: giantswarm/cloud-provider-cloud-director-app
-      version: 0.2.9
+      version: 0.2.9-d0437b6bea8531e199cab35e29cf9f751d48815d
```

Comment:
Please let me know if everything is fine regarding https://github.com/giantswarm/cloud-provider-cloud-director-app/pull/78#issuecomment-1905720470 then I can approve without worrying about it :D

Reply:
I want to merge this to `main` first in a separate PR.
https://github.com/giantswarm/cluster-cloud-director/pull/248